src/utils/Serial.js

```diff
@@ -252,6 +252,7 @@ class Serial {
   resetFourWayInterface = (esc) => this.fourWay.reset(esc);
   startFourWayInterface = () => this.fourWay.start();
   writeHex = (index, esc, hex, force, migrate, cbProgress) => this.fourWay.writeHex(index, esc, hex, force, migrate, cbProgress);
+  readAddress = (address, bytes, retries = 10) => this.fourWay.read(address, bytes, retries);
```

Comment:
The re-definitions here are to be seen as aliases or interface (keeping the same parameters ).
If you want to have an `readAddress` function, then please add it to `fourWay` and wrap the `read` function accordingly.

Reply:
Use the same function signature when re-declaring here:
```
read = (address, bytes, retries) => this.fourWay.read(address, bytes, retries);
```
just for sake of consistency. This needs some general re-factoring at one point.